impact_aware_cartesian_impedance_controller:
    type: franka_custom_controllers/ImpactAwareCartesianImpedanceController
    arm_id: panda
    joint_names:
        - panda_joint1
        - panda_joint2
        - panda_joint3
        - panda_joint4
        - panda_joint5
        - panda_joint6
        - panda_joint7
demonstration_cartesian_impedance_controller:
    type: franka_custom_controllers/DemonstrationCartesianImpedanceController
    arm_id: panda
    joint_names:
        - panda_joint1
        - panda_joint2
        - panda_joint3
        - panda_joint4
        - panda_joint5
        - panda_joint6
        - panda_joint7
        
